test/e2e/framework/pod/create.go

```diff
@@ -157,14 +157,20 @@ func MakePod(ns string, nodeSelector map[string]string, pvclaims []*v1.Persisten
 			RestartPolicy: v1.RestartPolicyOnFailure,
 		},
 	}
-	var volumeMounts = make([]v1.VolumeMount, len(pvclaims))
+	var volumeMounts = make([]v1.VolumeMount, 0)
+	var volumeDevices = make([]v1.VolumeDevice, 0)
 	var volumes = make([]v1.Volume, len(pvclaims))
 	for index, pvclaim := range pvclaims {
 		volumename := fmt.Sprintf("volume%v", index+1)
-		volumeMounts[index] = v1.VolumeMount{Name: volumename, MountPath: "/mnt/" + volumename}
+		if *pvclaim.Spec.VolumeMode == v1.PersistentVolumeBlock {
+			volumeDevices = append(volumeDevices, v1.VolumeDevice{Name: volumename, DevicePath: "/mnt/" + volumename})
```

Comment:
Instead of using "/mnt", how about something like "/devices". That probably would require changes to all the consumers.

Reply:
Actually, that may be too many changes for now, I take back that suggestion.

We actually already have [MakeSecPod](https://github.com/kubernetes/kubernetes/blob/f66869906d6fafd920f3e8e07020602b79eba88e/test/e2e/framework/pod/create.go#L183) which handles block devices already. What if we use that method instead?